projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -49,7 +49,7 @@ protected function __construct() {
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
 		add_action( 'wp_enqueue_scripts', array( $this, 'dequeue_scripts' ), 20 );
 		add_action( 'admin_enqueue_scripts', array( $this, 'dequeue_scripts' ), 20 );
-		add_action( 'rest_request_before_callbacks', array( $this, 'rest_api_init' ), 11 );
+		add_filter( 'rest_request_before_callbacks', array( $this, 'rest_api_init' ), 11, 3 );
```

Comment:
Nitpick: Since we're using only the first parameter of the filter, we don't that `3` at the end.
```suggestion
		add_filter( 'rest_request_before_callbacks', array( $this, 'rest_api_init' ), 11 );
```